pkg/s3repository/repository_protected_entity.go

```diff
@@ -602,7 +604,7 @@ func (this *ProtectedEntity) cleanupOnAbortedUpload(ctx *context.Context) {
 		log.Infof("The context was canceled during copy of pe %v, proceeding with cleanup", peInfo.GetName())
 		log.Debugf("Attempting to delete any uploaded snapshots for %v", this.peinfo.GetID())
 		// New context or else downstream "withContext" calls will error out.
-		status, err := this.DeleteSnapshot(context.Background(), this.peinfo.GetID().GetSnapshotID())
+		status, err := this.DeleteSnapshot(context.Background(), this.peinfo.GetID().GetSnapshotID(), nil)
```

Comment:
We're passing in an empty map everywhere else.  Let's pass in the empty map here as well

Reply:
Done. Here and elsewhere.